airflow-core/src/airflow/models/dag.py

```diff
@@ -522,8 +524,7 @@ def get_paused_dag_ids(dag_ids: list[str], session: Session = NEW_SESSION) -> se
             .where(DagModel.dag_id.in_(dag_ids))
         )
 
-        paused_dag_ids = {paused_dag_id for (paused_dag_id,) in paused_dag_ids}
-        return paused_dag_ids
+        return {paused_dag_id for (paused_dag_id,) in paused_dag_ids}
```

Comment:
Change `session.execute` to `session.scalars` and we can change the below to simply `return set(paused_dag_ids)`